WordPress/src/main/java/org/wordpress/android/ui/photopicker/MediaPickerLauncher.kt

```diff
@@ -212,6 +212,32 @@ class MediaPickerLauncher @Inject constructor(
         }
     }
 
+    fun showAudioFilePicker(activity: Activity, canMultiselect: Boolean = false) {
+        if (consolidatedMediaPickerFeatureConfig.isEnabled()) {
```

Comment:
I am not sure if this approach is fine. What I did here was I duplicated the `showFilePicker` method because I need the `allowTypes` to be different for the file and audio pickers. At first, I thought of making this property a default one in the method params but this doesn't work with Java. Is this behavior okay? Or should I do some form of method override? Let me know.

Reply:
Also, I didn't bother to bring the Normal Picker behavior up to speed here because it seems like the consolidated media picker will be the only one in use for the near future so I thought I would waste time building it out and there's a high possibility we won't do a rollback. Was that the right judgment call? ? I did make modifications in the past to support the File Block's file selection behavior on the normal and consolidated picker hence my question.

Reply:
This is great insight @develric I am going to do this implementation and then share my findings with you.

Reply:
Done ff71990 3ca76e1